docs/apis/plugintypes/quizaccess/_examples/override_rule.php

```diff
@@ -0,0 +1,48 @@
+use mod_quiz\local\access_override_rule_base;
+use MoodleQuickForm;
+use context_module;
+use context;
+
+class override_rule extends access_override_rule_base {
+
```

Comment:
Please pass this file through the code style checker and fix it.

It's worth noting tha twe also tend to further reduce line length limits to around 80 characters in docs because the way that code scrolls left-to-right makes it hard to view things in context.